*** tests/A-003-007-M-ok.out	Wed Nov 26 23:23:51 2014
--- tests/A-003-007-M-ok.outhyp	Tue Dec  2 18:36:33 2014
***************
*** 23,32 ****
  10 - Mudar permissões de entrada
  11 - Mudar dono de entrada
  0 - Sair
! Escolha uma opção: d - vader 32 .
! d - root 40 ..
! - w vader 44 calendar
! d w vader 16 friends
  Menu de Shell
  1 - Listar
  2 - Listar entrada
--- 23,30 ----
  10 - Mudar permissões de entrada
  11 - Mudar dono de entrada
  0 - Sair
! Escolha uma opção: d - root 32 root
! d - root 56 home
  Menu de Shell
  1 - Listar
  2 - Listar entrada
