*** tests/A-006-003-M-ok.out	Wed Nov 26 23:23:51 2014
--- tests/A-006-003-M-ok.outhyp	Tue Dec  2 18:36:34 2014
***************
*** 14,51 ****
  1 - Criar utilizador
  2 - Listar utilizadores
  0 - Sair
! Escolha uma opção: Identificador do utilizador: Nome do utilizador: Menu Utilizador
  1 - Criar utilizador
  2 - Listar utilizadores
  0 - Sair
! Escolha uma opção: Identificador do utilizador: Nome do utilizador: Menu Utilizador
  1 - Criar utilizador
  2 - Listar utilizadores
  0 - Sair
! Escolha uma opção: Identificador do utilizador: Nome do utilizador: Menu Utilizador
  1 - Criar utilizador
  2 - Listar utilizadores
  0 - Sair
! Escolha uma opção: Identificador do utilizador: Nome do utilizador: Menu Utilizador
  1 - Criar utilizador
  2 - Listar utilizadores
  0 - Sair
! Escolha uma opção: Identificador do utilizador: Nome do utilizador: Menu Utilizador
  1 - Criar utilizador
  2 - Listar utilizadores
  0 - Sair
! Escolha uma opção: Identificador do utilizador: Nome do utilizador: Menu Utilizador
  1 - Criar utilizador
  2 - Listar utilizadores
  0 - Sair
! Escolha uma opção: a:a:/home/a
! aa:a:/home/aa
! d:d:/home/d
! r:r:/home/r
! root:Super User:/home/root
! rr:rr:/home/rr
! z:z:/home/z
! Menu Utilizador
  1 - Criar utilizador
  2 - Listar utilizadores
  0 - Sair
--- 14,45 ----
  1 - Criar utilizador
  2 - Listar utilizadores
  0 - Sair
! Escolha uma opção: Nome do utilizador: Identificador do utilizador: Menu Utilizador
  1 - Criar utilizador
  2 - Listar utilizadores
  0 - Sair
! Escolha uma opção: Nome do utilizador: Identificador do utilizador: Menu Utilizador
  1 - Criar utilizador
  2 - Listar utilizadores
  0 - Sair
! Escolha uma opção: Nome do utilizador: Identificador do utilizador: Menu Utilizador
  1 - Criar utilizador
  2 - Listar utilizadores
  0 - Sair
! Escolha uma opção: Nome do utilizador: Identificador do utilizador: Menu Utilizador
  1 - Criar utilizador
  2 - Listar utilizadores
  0 - Sair
! Escolha uma opção: Nome do utilizador: Identificador do utilizador: Menu Utilizador
  1 - Criar utilizador
  2 - Listar utilizadores
  0 - Sair
! Escolha uma opção: Nome do utilizador: Identificador do utilizador: Criar utilizador: Operação inválida: O utilizador 'a' já existe.
! Menu Utilizador
  1 - Criar utilizador
  2 - Listar utilizadores
  0 - Sair
! Escolha uma opção: Menu Utilizador
  1 - Criar utilizador
  2 - Listar utilizadores
  0 - Sair
