*** tests/A-014-004-M-ok.out	Wed Nov 26 23:23:51 2014
--- tests/A-014-004-M-ok.outhyp	Tue Dec  2 18:36:39 2014
***************
*** 40,47 ****
  10 - Mudar permissões de entrada
  11 - Mudar dono de entrada
  0 - Sair
! Escolha uma opção: d - yoda 16 .
! d - root 40 ..
  Menu de Shell
  1 - Listar
  2 - Listar entrada
--- 40,47 ----
  10 - Mudar permissões de entrada
  11 - Mudar dono de entrada
  0 - Sair
! Escolha uma opção: d - yoda 32 yoda
! d - root 56 home
  Menu de Shell
  1 - Listar
  2 - Listar entrada
