*** tests/A-012-002-M-ok.out	Wed Nov 26 23:23:51 2014
--- tests/A-012-002-M-ok.outhyp	Tue Dec  2 18:36:37 2014
***************
*** 27,33 ****
  10 - Mudar permissões de entrada
  11 - Mudar dono de entrada
  0 - Sair
! Escolha uma opção: Indique o nome do ficheiro: Insira a linha de texto a acrescentar ao ficheiro: Adicionar linha a ficheiro: Operação inválida: A entrada 'public' não é um ficheiro.
  Menu de Shell
  1 - Listar
  2 - Listar entrada
--- 27,33 ----
  10 - Mudar permissões de entrada
  11 - Mudar dono de entrada
  0 - Sair
! Escolha uma opção: Indique o nome da entrada: Insira a linha de texto a acrescentar ao ficheiro: Adicionar linha a ficheiro: Operação inválida: A entrada 'public' não é um ficheiro.
  Menu de Shell
  1 - Listar
  2 - Listar entrada
