*** tests/A-014-007-M-ok.out	Wed Nov 26 23:23:51 2014
--- tests/A-014-007-M-ok.outhyp	Tue Dec  2 18:36:40 2014
***************
*** 66,73 ****
  10 - Mudar permissões de entrada
  11 - Mudar dono de entrada
  0 - Sair
! Escolha uma opção: d w vader 24 .
! d - vader 40 ..
  - - vader 23 filePriv
  Menu de Shell
  1 - Listar
--- 66,73 ----
  10 - Mudar permissões de entrada
  11 - Mudar dono de entrada
  0 - Sair
! Escolha uma opção: d w vader 40 public
! d - vader 56 vader
  - - vader 23 filePriv
  Menu de Shell
  1 - Listar
