*** tests/A-011-004-M-ok.out	Wed Nov 26 23:23:51 2014
--- tests/A-011-004-M-ok.outhyp	Tue Dec  2 18:36:37 2014
***************
*** 27,33 ****
  10 - Mudar permissões de entrada
  11 - Mudar dono de entrada
  0 - Sair
! Escolha uma opção: Indique o nome do ficheiro: This is a file
  
  Menu de Shell
  1 - Listar
--- 27,33 ----
  10 - Mudar permissões de entrada
  11 - Mudar dono de entrada
  0 - Sair
! Escolha uma opção: Indique o nome da entrada: This is a file
  
  Menu de Shell
  1 - Listar
